lib/rails_extensions/read_only.rb

```diff
@@ -1,7 +0,0 @@
-if Rails.configuration.read_only
-  module ActiveRecord::Base
-    def readonly?
```

Comment:
I couldn't find any reference to this method outside of the definition.  Looks like everything was just checking the environment variable.